Intersect.Client/MonoGame/Graphics/MonoRenderer.cs

```diff
@@ -739,7 +739,7 @@ public override List<string> GetValidVideoModes()
 
             var allowedResolutions = new[]
             {
-                new Resolution(800, 600),
+                new Resolution(800),
```

Comment:
Don't remove the `, 600`

Reply:
whops 😮 🤣

Reply:
oh, this is why:

![imagen](https://user-images.githubusercontent.com/17498701/210284530-67ef0e07-2383-4c78-97b1-363b8e5feb52.png)

i will restore it back anyways.